tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/AutoTunerSuite.scala

```diff
@@ -99,7 +125,8 @@ class AutoTunerSuite extends FunSuite with BeforeAndAfterEach with Logging {
 
   test("Load cluster properties with CPU cores 0") {
     val dataprocWorkerInfo = buildWorkerInfoAsString(None, Some(0))
-    val autoTuner: AutoTuner = AutoTuner.buildAutoTunerFromProps(dataprocWorkerInfo, None)
+    val autoTuner: AutoTuner = AutoTuner.buildAutoTunerFromProps(dataprocWorkerInfo,
+      getMockInfoProvider)
```

Comment:
only concern here is that the getMockInfoProvider is not used and its not completely obvious here at first look.  Can we perhaps create one that is UnusedProvider

Reply:
I changed the name of the method to be "getUnusedProvider" which constructs an object of `AppSummaryInfoBaseProvider`